src/util/log/fd_log.h

```diff
@@ -142,6 +142,8 @@
 
 #include "../env/fd_env.h"
 
+#define FD_LOG_BUF_SZ               4*4096
```

Comment:
This probably belongs in the log implementation itself (as per previous discussions, other implementations of this API might use different sizes and what not ... so details like this probably shouldn't be exposed or should be exposed in via an explicit API to query the implementation).


Also, to avoid Tom-Foolery with macros and token pasting and silent conversions), this should be something like `#define FD_LOG_BUF_SZ (4UL*4096UL)`.